index.js

```diff
@@ -1,27 +1,32 @@
 // This will check if the node version you are running is the required
 // Node version, if it isn't it will throw the following error to inform
 // you.
-if (Number(process.version.slice(1).split(".")[0]) < 8) throw new Error("Node 8.0.0 or higher is required. Update Node on your system.");
+if (Number(process.version.slice(1).split(".")[0]) < 14) throw new Error("Node 14.0.0 or higher is required. Update Node on your system.");
```

Comment:
Actually, while validating this, Discord.JS V12.5.1 only requires Node version 12.0.0 or higher. While yes, we should require users to use an LTS version - we shouldn't set it as the requirement.

Reply:
Ok